reporters.go

```diff
@@ -6,19 +6,19 @@ import (
 	"strings"
 )
 
-// reportInvalidArguments reports invalid agument to fail the test
+// reportInvalidArguments reports invalid argument to fail the test
 func reportInvalidArguments(testReporter TestReporter, functionType reflect.Type, arguments []interface{}) {
-	real := make([]string, functionType.NumIn())
+	realArgTypes := make([]string, functionType.NumIn())
 	for i := 0; i < functionType.NumIn(); i++ {
 		if isVariadicArgument(functionType, i) {
-			real[i] = fmt.Sprintf("...%v", toFriendlyName(functionType.In(i).Elem()))
+			realArgTypes[i] = fmt.Sprintf("...%v", toFriendlyName(functionType.In(i).Elem()))
 			continue
 		}
 
-		real[i] = toFriendlyName(functionType.In(i))
+		realArgTypes[i] = toFriendlyName(functionType.In(i))
 	}
 
-	testReporter.Errorf("mocka: expected arguments of type (%v), but received (%v)", strings.Join(real, ", "), strings.Join(mapToTypeName(arguments), ", "))
+	testReporter.Errorf("mocka: expected arguments of type (%v), but received (%v)", strings.Join(realArgTypes, ", "), strings.Join(mapToTypeName(arguments), ", "))
```

Comment:
I don't see the value in this type of change. This one and the one below it are small enough that they will not cause any other side effects so we can keep them. However, I would like to keep changes to mocka small and focused. That way we don't introduce bugs or anything by making unnecessary changes like renames.

Reply:
fair enough. this was an issue uncovered by the linter which is why I did it as part of this PR